HelpSource/Classes/Server.schelp

```diff
@@ -623,6 +623,15 @@ an optional instance of link::Classes/Condition:: used for evaluating this.
 argument:: ... args
 one or more valid OSC messages.
 
+method:: addr
+Returns the ip and TCP/UDP port being used by the specified server
+code::
+~myLocalServer=Server.local;
+~myLocalServer.addr;
+~myInternalServer=Server.internal;
+~myInternalServer.addr;
+::
```

Comment:
@snappizz Does internal server still exist? I'm confused, I thought we got rid of it..